src/components/AboutUsPage.js

```diff
@@ -0,0 +1,159 @@
+import styled from 'styled-components';
+import Card from 'react-bootstrap/Card';
+import { useNavigate } from 'react-router-dom';
+
```

Comment:
To use a local image, you'd have to import it first like this:
`import samplePortrait from './component-images/sample-portrait.png';`